redash/handlers/api.py

```diff
@@ -69,6 +70,11 @@ def json_representation(data, code, headers=None):
 api.add_org_resource(QueryRefreshResource, '/api/queries/<query_id>/refresh', endpoint='query_refresh')
 api.add_org_resource(QueryResource, '/api/queries/<query_id>', endpoint='query')
 
+api.add_org_resource(AccessListResource, '/api/access/<object_type>/<object_id>', endpoint='list_access')
+api.add_org_resource(AccessGrantResource, '/api/access/<object_type>/<object_id>', endpoint='grant_access')
+api.add_org_resource(AccessRevokeResource, '/api/access/<object_type>/<object_id>', endpoint='revoke_access')
```

Comment:
After looking at the usage, I think we should change this to be: `/api/<object_type>/<object_id>/acl`, and `object_type` should be `queries` and `dashboards` to be consistent with the rest of the API. In the handler itself, we can map it to `Query` and `Dashboard` (or the object class).